apps/parsley/src/hooks/useSections/index.ts

```diff
@@ -31,8 +31,10 @@ export type ToggleFunctionSection = (props: {
 
 export interface UseSectionsResult {
   sectionData: SectionData | undefined;
+  toggleAllCommandsInFunction: (functionID: string, isOpen: boolean) => void;
   toggleCommandSection: ToggleCommandSection;
```

Comment:
Sorry, not exactly in this PR but I noticed an extraneous animation being applied to the carets on the side when toggling

https://github.com/user-attachments/assets/69fedd41-bb31-4787-8dc4-1dde4cca9f2e

Reply:
It's the long log you sent me :) https://parsley-beta.corp.mongodb.com/evergreen/spruce_ubuntu2204_e2e_test_patch_fa2cf392a15e9ab48df7419d3f91baa19753f85c_6553dbb83e8e86081627dd4a_23_11_14_20_42_33/0/task?bookmarks=0,108340

Reply:
It's the long log you sent me :) https://parsley-beta.corp.mongodb.com/evergreen/spruce_ubuntu2204_e2e_test_patch_fa2cf392a15e9ab48df7419d3f91baa19753f85c_6553dbb83e8e86081627dd4a_23_11_14_20_42_33/0/task?bookmarks=0,108340

Reply:
Thank you, tracking here: https://jira.mongodb.org/browse/DEVPROD-9783